daml-lf/spec/value.rst

```diff
@@ -155,7 +155,7 @@ This table lists every version of this specification in ascending order
 +--------------------+-----------------+
 |                  3 |      2019-02-14 |
 +--------------------+-----------------+
-|                  4 |      2019-03-27 |
+|                  4 |      2019-04-10 |
```

Comment:
3-27 might still be more accurate; at your discretion.

Reply:
I was not sure. I follow your opinion.